spec/support/helpers/events.rb

```diff
@@ -0,0 +1,9 @@
+module Helpers
```

Comment:
This is already a top level module in rails, and _I think_ will include automatically include all modules below it in Controllers and Views - potentially causing issues for specs in the future.

I'd suggest renaming this to something like `SpecHelpers` ?

Reply:
Oh ok, I was going off the RSpec docs but happy to rename it:

https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module